### PR TITLE
Release notes v5.1.0

### PR DESCRIPTION
**Pull request by lcheunglci**
Add release notes for 5.1 GA Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -321,13 +340,14 @@ This update brings the below changes over the previous release:
...
+- Various code improvements [#902](https://github.com/dotnet/SqlClient/pull/902) [#925](https://github.com/dotnet/SqlClient/pull/925) [#933](https://github.com/dotnet/SqlClient/pull/933) [#934](https://github.com/dotnet/SqlClient/pull/934) [#1024](https://github.com/dotnet/SqlClient/pull/1024) [#1057](https://github.com/dotnet/SqlClient/pull/1057) [#1122](https://github.com/dotnet/SqlClient/pull/1122) [#1133]((<https://github.com/dotnet/SqlClient/pull/1133>)) [#1134](https://github.com/dotnet/SqlClient/pull/1134) [#1141](https://github.com/dotnet/SqlClient/pull/1141) [#1187](https://github.com/dotnet/SqlClient/pull/1187) [#1188](https://github.com/dotnet/SqlClient/pull/1188) [#1223](https://github.com/dotnet/SqlClient/pull/1223) [#1225](https://github.com/dotnet/SqlClient/pull/1225)  [#1226](https://github.com/dotnet/SqlClient/pull/1226)
```
Comment (ErikEJ) [inline CHANGELOG.md:350]: Brackets added in error?

File: release-notes/5.1/5.1.0.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 5.1.0 released 19 January 2023
+
+This update includes the following changes over the 5.0 release:
+
+### Breaking changes
+
+- Add support for .NET 6.0 and Dropped support for .NET Core 3.1. [#1704](https://github.com/dotnet/SqlClient/pull/1704) [#1823](https://github.com/dotnet/SqlClient/pull/1823)
```
Comment (ErikEJ) [inline release-notes/5.1/5.1.0.md:9]: "Added" "dropped"

File: CHANGELOG.md (hunk 1 above)
Comment (lcheunglci) [reply CHANGELOG.md]: Thanks, I ran the `Fix all supported markdownlint violations in the document` from the `quick fix` option from the `learn-markdown` extension but missed it in my diff.

File: release-notes/5.1/5.1.0.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+- Changed the `SqlConnectionEncryptOption` string parser to public. [#1771](https://github.com/dotnet/SqlClient/pull/1771)
+- Converted `ExecuteNonQueryAsync` to use async context object. [#1692](https://github.com/dotnet/SqlClient/pull/1692)
+- Code health improvements [#1849](https://github.com/dotnet/SqlClient/pull/1849) [#1812](https://github.com/dotnet/SqlClient/pull/1812) [#1520](https://github.com/dotnet/SqlClient/pull/1520) [#1604](https://github.com/dotnet/SqlClient/pull/1604) [#1598](https://github.com/dotnet/SqlClient/pull/1598) [#1595](https://github.com/dotnet/SqlClient/pull/1595) [#1443](https://github.com/dotnet/SqlClient/pull/1443)
+
+## New features
+
+### Server Certificate Support
+
+The default value of the `ServerCertificate` connection setting is an empty string.  When `Encrypt` is set to `Mandatory` or `Strict`, `ServerCertificate` can be used to specify a path on the file system to a certificate file to match against the SQL Server's TLS/SSL certificate. For this to be valid, the certificate specified must be an exact match. The accepted certificate formats are `PEM`, `DER`, and `CER`. Here is an usage example:
```
Comment (David-Engel) [inline release-notes/5.1/5.1.0.md:44]:
```suggestion
The default value of the `ServerCertificate` connection setting is an empty string.  When `Encrypt` is set to `Mandatory` or `Strict`, `ServerCertificate` can be used to specify a path on the file system to a certificate file to match against the SQL Server's TLS/SSL certificate. For this to be valid, the certificate specified must be an exact match. The accepted certificate formats are `PEM`, `DER`, and `CER`. Here is an example:
```

File: release-notes/5.1/5.1.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # Microsoft.Data.SqlClient 5.1 Releases
 
+The following Microsoft.Data.SqlClient 5.1 stable releases have been shipped:
+
+| Release Date | Version | Notes |
+| :-- | :-- | :--: |
+| 2022/01/19 | 5.1.0 | [release notes](5.1.0.md) |
```
Comment (Kaur-Parminder) [inline release-notes/5.1/5.1.md:7]:
```suggestion
| 2023/01/19 | 5.1.0 | [release notes](5.1.0.md) |
```

File: release-notes/5.1/5.1.0.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+- Reverted "Excluding unsupported TLS protocols" for issue [#1151](https://github.com/dotnet/SqlClient/issues/1151) (i.e. removed `Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS`) by adding support for TLS 1.3. [#1824](https://github.com/dotnet/SqlClient/issues/1824)
+- Changed the `SqlConnectionEncryptOption` string parser to public. [#1771](https://github.com/dotnet/SqlClient/pull/1771)
+- Converted `ExecuteNonQueryAsync` to use async context object. [#1692](https://github.com/dotnet/SqlClient/pull/1692)
+- Code health improvements [#1849](https://github.com/dotnet/SqlClient/pull/1849) [#1812](https://github.com/dotnet/SqlClient/pull/1812) [#1520](https://github.com/dotnet/SqlClient/pull/1520) [#1604](https://github.com/dotnet/SqlClient/pull/1604) [#1598](https://github.com/dotnet/SqlClient/pull/1598) [#1595](https://github.com/dotnet/SqlClient/pull/1595) [#1443](https://github.com/dotnet/SqlClient/pull/1443)
```
Comment (Kaur-Parminder) [inline release-notes/5.1/5.1.0.md:38]: PR 1867? https://github.com/dotnet/SqlClient/pull/1867

**Review by Kaur-Parminder — APPROVED** [review]

**Review by DavoudEshtehari — APPROVED** [review]

File: release-notes/5.1/5.1.0.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+This update includes the following changes over the 5.0 release:
+
+### Breaking changes
+
+- Dropped support for .NET Core 3.1. [#1704](https://github.com/dotnet/SqlClient/pull/1704) [#1823](https://github.com/dotnet/SqlClient/pull/1823)
+
+### Added
+
+- Added support for `DateOnly` and `TimeOnly` for `SqlParameter` value and `GetFieldValue`. [#1813](https://github.com/dotnet/SqlClient/pull/1813)
```
Comment (DavoudEshtehari) [inline release-notes/5.1/5.1.0.md:13]:
```suggestion
- Added support for .NET 6.0. [#1704](https://github.com/dotnet/SqlClient/pull/1704)
- Added support for `DateOnly` and `TimeOnly` for `SqlParameter` value and `GetFieldValue`. [#1813](https://github.com/dotnet/SqlClient/pull/1813)
```

**Review by JRahnama — APPROVED** [review]

**Review by David-Engel — APPROVED** [review]

**Comment by ErikEJ** [comment]
Congrats on a great release!

**Comment by lcheunglci** [comment]
Thanks everyone for your help and contributions!

**Comment by DavoudEshtehari** [comment]
Thank you @ErikEJ. A big kudos to all kind contributors. ❤️

**Pull request merged**